app/src/androidTest/java/com/sdp/movemeet/view/navigation/NavigationTest.java

```diff
@@ -53,12 +67,16 @@
     //@Rule
     //public ActivityScenarioRule<MainActivity> testRule = new ActivityScenarioRule<>(MainActivity.class)
 
-    @Test
-    public void truc(){}
-    /*@Before
+
+    @Before
     public void signIn(){
+        Navigation.profileField = false;
         MainActivity.enableNav = true;
         UploadActivityActivity.enableNav = true;
+        ActivityDescriptionActivity.enableNav = true;
+        ChatActivity.enableNav = true;
```

Comment:
Do you really want to enable the navigation to the chat knowing that, as we observed so far, any navigation to the chat may induce a "`java.lang.RuntimeException: Uncaught exception in Firebase Database runloop (19.7.0). If you are not already on the latest version of the Firebase SDKs, try updating your dependencies`" error? Is it required?

Reply:
In any case, it is not this line that could create the error. It just allows you to specify that at the instantiation of the chat activity, the menu will be created. So in order to have a menu from the chat, this line is necessary.